source/common/settings/user-profile.rst

```diff
@@ -7,4 +7,15 @@ User Profile
 ============
 You can update your user profile from the **Account** tab on the **Settings** page. To access this page, click your user name or gravatar in the lower portion of the navigation pane.
 
-Update your name, email address or password and click **Save Changes**.
\ No newline at end of file
+Update your name, email address or password and click **Save Changes**.
+
+Delete your account
+-------------------
+
+You can delete your account at any time from the **Codio > Account** menu item and going to the **Account** tab
+
+   .. image:: /img/delete.png
+      :alt: Delete account
+      
+
```

Comment:
**Warning:** If you delete your account it cannot be retrieved nor can its contents be restored. To archive your data for potential later usage, export your projects or push them to a remote depository.